repair/row_level.cc

```diff
@@ -695,7 +731,22 @@ class repair_meta {
     sink_source_for_put_row_diff _sink_source_for_put_row_diff;
     tracker_link_type _tracker_link;
     row_level_repair* _row_level_repair_ptr;
+    std::vector<repair_node_state> _all_node_states;
 public:
+    std::vector<repair_node_state>& all_nodes() {
+        return _all_node_states;
+    }
+    void set_repair_state(repair_state state, gms::inet_address node) {
+        for (auto& ns : all_nodes()) {
```

Comment:
Won't this be slow in large clusters? A node can have has many peers as there are tokens.

Reply:
It only depends on the RF not the number of nodes in cluster. Also set_repair_state is called on the stop code path, which is a slow path.

Reply:
Ah, this is per-vnode and not per-repair?

Reply:
Yes.